File: 2133/2133.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var checkIfEveryRowAndColumnContai = function(root) {
    if (!root) return 0;
    return 1 + Math.max(checkIfEveryRowAndColumnContai(root.left), checkIfEveryRowAndColumnContai(root.right));
};

File: 2133/2133.go
func checkIfEveryRowAndColumnContai(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := checkIfEveryRowAndColumnContai(root.Left)
	r := checkIfEveryRowAndColumnContai(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}